.github/workflows/main.yml

```diff
@@ -0,0 +1,87 @@
+name: OWTF CI
+
+on:
+  push:
+    branches: [master, develop]
+  pull_request:
+    branches: [master, develop]
+
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        python-version: [3.9]
+        node-version: [15.0]
+    services:
+      db:
+        image: postgres
+        env:
+          POSTGRES_USER: owtf_db_user
+          POSTGRES_PASSWORD: jgZKW33Q+HZk8rqylZxaPg1lbuNGHJhgzsq3gBKV32g=
```

Comment:
I know this is for tests, but we can store this secret as a Github Actions secret (so that it is not checked in git)

Reply:
Sure, can you create the secrets and share them so that I can add them here.